nucypher/blockchain/eth/sol/source/contracts/Issuer.sol

```diff
@@ -160,6 +163,16 @@ contract Issuer is Upgradeable {
         currentSupply2 = currentSupply2.sub(_amount);
     }
 
+    /**
+    * @notice Burn sender's tokens. Amount of tokens will be returned for future minting
+    * @param _value Amount to burn
+    **/
+    function burn(uint256 _value) public {
+        token.safeTransferFrom(msg.sender, address(this), _value);
```

Comment:
Alternatively, or in addition, we could limit the use of this function to participants that end up with less than the min stake size